conf/pxe_cluster/grub.cfg

```diff
@@ -0,0 +1,24 @@
+# Copyright 2017 ARICENT HOLDINGS LUXEMBOURG SARL and Cable Television Laboratories, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# The file Grub.cfg is used for UEFI Installers
+# Defines the ip/host for the Build Server that contains the
+# http and tftp servers containing the netboot image
+#
+
+menuentry "Install Ubuntu" {
+set gfxpayload=keep
+linux ubuntu-installer/amd64/linux gfxpayload=800x600x16,800x600 live-installer/net-image=http://172.16.141.17/ubuntu/install/filesystem.squashfs console=ttyS1,115200 console=ttyS0,115200 console=tty ramdisk_size=16432 root=/dev/rd/0 rw --- auto=true url=http://172.16.141.17/ubuntu/preseed/hwe-ubuntu-server.seed
```

Comment:
What is the 172.16.141.17 ip mean to be used for?

Reply:
its a place holder for the web/dhcp server to pull the seed file from.

Reply:
it gets replaced during -p phase